lib/Doctrine/Migrations/Tools/Console/Command/ExecuteCommand.php

```diff
@@ -90,20 +94,27 @@ protected function configure() : void
 
     public function execute(InputInterface $input, OutputInterface $output) : ?int
     {
-        $version        = $input->getArgument('version');
+        $versions       = $input->getArgument('versions');
         $timeAllQueries = (bool) $input->getOption('query-time');
         $dryRun         = (bool) $input->getOption('dry-run');
         $path           = $input->getOption('write-sql');
         $direction      = $input->getOption('down') !== false
             ? Direction::DOWN
             : Direction::UP;
 
-        $version = $this->migrationRepository->getVersion($version);
+        $versions = array_map(
+            function (string $version) {
```

Comment:
Please add a return type hint

Reply:
And making it `static` would be nice too :-)